cscope.py

```diff
@@ -286,6 +282,7 @@ def run_cscope(self, mode, word):
         if (self.platform == "windows"):
             popen_arg_list["creationflags"] = 0x08000000
             popen_arg_list["stdin"] = subprocess.PIPE
+            popen_arg_list["universal_newlines"] = True
```

Comment:
It looks like this is simply a better way of doing this, regardless of platform, so can you move this up to always be in popen_arg_list, maybe after line 280?